tests/test.rfb.js

```diff
@@ -3884,6 +3892,26 @@ describe('Remote Frame Buffer protocol client', function () {
                                                                  50, 70, 0x0);
             });
 
+            it('should send extended pointer event when server supports extended pointer events', function () {
+                client._extendedPointerEventSupported = true;
+                sendMouseButtonEvent(50, 70, true, 0x10, client);
+
+                expect(extendedPointerEvent).to.have.been.calledOnceWith(client._sock,
+                                                                         50, 70, 0x100);
+            });
+
+            it('should send normal pointer event when server supports does not extended pointer events', function () {
```

Comment:
Should also have a separate test that shows that buttons are filtered out for old servers.